File: sample-aac/src/main/java/sky/example/MainActivity.java
package sky.example;

import android.os.Bundle;

import sk.SKActivity;
import sk.SKActivityBuilder;
import sky.example.fragment.HelloFragment;

/**
 * @author sky
 * @version 1.0 on 2018-04-28 上午10:07
 * @see MainActivity
 */
public class MainActivity extends SKActivity<MainBiz> {
	@Override protected SKActivityBuilder build(SKActivityBuilder skBuilder) {
		skBuilder.layoutId(R.layout.activity_main);
		return skBuilder;
	}

	@Override protected void initData(Bundle savedInstanceState) {
		getSupportFragmentManager().beginTransaction().add(R.id.ll_main,HelloFragment.getInstance(), "aaaa").commit();

	}
}

File: sample-kotlin/src/main/java/sky/sample/MainActivity.kt
package sky.sample

import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*
import sky.core.SKYActivity
import sky.core.SKYBuilder

/**
 * @author sky
 * @version 1.0 on 2017-12-06 下午3:50
 * @see MainActivity
 */
open class MainActivity : SKYActivity<MainBiz>() {

    override fun build(initialSKYBuilder: SKYBuilder): SKYBuilder {
        initialSKYBuilder.layoutId(R.layout.activity_main)
        return initialSKYBuilder
    }

    override fun initData(savedInstanceState: Bundle?) {
        button2.setOnClickListener { biz().a() }
        button3.setOnClickListener { biz().b("aa",1) }
    }

}
